### ISSUE TITLE
Windows that demand focus will minimize fullscreen applications

### ISSUE THREAD

**Issue by shoober420** [baskerville/bspwm#607]
When I was on Steam playing a game fullscreen, I got a message from a friend. This opens a separate floating window. Instead of the window remaining in the background, it forces the game to windowed mode with the dimensions all messed up.

**Comment by shoober420** [comment]
For now, I've simply moved the Steam chat window to another desktop so it doesn't minimize the current program on the desktop with the fullscreen program.

**Comment by msteen** [comment]
I have not tested it, but should the setting `ignore_ewmh_focus` to true, not solve this issue for you?

**Comment by danikron** [comment]
This seems to be generally true, regardless of ignore_ewmh_focus. If any floating node is opened on a desktop containing a fullscreen node then the fullscreen flag will be unset. Also, if a floating node is in focus over a tiled node and then the fullscreen flag is set for the tiled node then the floating node will be covered even though it retains focus.

**Comment by shoober420** [comment]
Im going to close this, as I dont think this is actually a bug.

**Comment by olzraiti** [comment]
On my Ubuntu machine I'm running bspwm 0.9.3, and there a fullscreen window will stay fullscreen even if some window other floating window is opened.

On my other machine I'm running `0.9.10-9-g853bb73`, and opening a floating window will make the fullscreen window exit fullscreen mode. `ignore_ewmh_focus` doesn't have any effect on this.

So this behaviour has changed - doesn't mean that it's a bug but it's a breaking change and possibly an unintentional change anyway. I liked the old behavior more (most usual use case for me is having a window fullscreened, and I bring a quake-like dropdown terminal into focus, and I'd expect the fullscreened window to stay that way).

I'm just one user and someone else might like the new behaviour, maybe it could be made configurable?

**Comment by KaosBunny** [comment]
Yeah, I'm also having this issue. It's incredibly frustrating. Is there still no way to disable this behaviour? If I have something set to fullscreen, I want it to _stay_ fullscreen.

**Comment by brongulus** [comment]
Can confirm, the issue still persists even with all these config settings. Version: 0.9.10

**Comment by negativems** [comment]
Still have this problem

**Comment by ortango** [comment]
>In each layer, the window are orderered as follow: tiled & pseudo-tiled < floating < fullscreen.

`neutralize_occluding_windows()` will unfullscreen a window so that you can see the window at the lower level (`stack_level()`). you could use an external rules script to check for a local fullscreen window and either set the incoming floating window as `layer=above` or `focus=off`.
